include/net/lwm2m.h

```diff
@@ -311,6 +312,99 @@ struct coap_block_context *lwm2m_firmware_get_block_context();
 #endif
 #endif
 
+
+#if defined(CONFIG_LWM2M_SWMGMT_OBJ_SUPPORT)
+
+/**
+ * @breif Set callback to handle software activation requests
```

Comment:
It's "brief" :-)

Reply:
Normally, spelling mistakes are not a big deal, but in this case the @brief tag is used by tools to automatically produce documentation.

Reply:
I thought brief was some interpretation of brief in Swedish